### ISSUE TITLE
Chat -PDF file is not visible in the chat conversation, but it can be downloaded.

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#12275]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to URL https://staging.new.expensify.com/
2. Log in with any account
2. Navigate to any conversation
3. Tap on Plus button - Add attachment
4. Select pdf-file
5. Send this file

**PDF file** 
[privat24_receipt.pdf](https://github.com/Expensify/App/files/9892046/privat24_receipt.pdf)

## Expected Result:
User should see the PDF file in conversation 

## Actual Result:
PDF file is not visible in the chat, but it can be downloaded.

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- Mobile Web

**Version Number:** 1.2.21.4

**Reproducible in staging?:** Yes

**Reproducible in production?:** No

**Email or phone of affected tester (no customers):** 

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/198736232-3fe0b623-8558-4f00-9926-45fa7b35a51c.mp4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause - Internal Team

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mountiny** [comment]
Tested with @Julesssss and we canot repro in staging

<img width="376" alt="image" src="https://user-images.githubusercontent.com/36083550/199036143-43e577db-3acd-4b66-886f-73e608275e85.png">


Work fine on phone too

**Comment by Julesssss** [comment]
This is not an App issue, it's a web regression introduced here. Given that most pdf files work correctly, I agree with removing the deploy blocker label

**Comment by Julesssss** [comment]
It seems to be introduced by[ this Web PR](https://github.com/Expensify/Web-Expensify/pull/35274), as this is now reproducable on App prod.

**Comment by mountiny** [comment]
cc @youssef-lr would you be able to look into this one?

**Comment by youssef-lr** [comment]
It looks like the upload fails when the `fileMimeType` is `applicatoin/octet-steam` instead of `application/pdf` due to this change I added [here](https://github.com/Expensify/Web-Expensify/blob/main/lib/ReportUtils.php#L348) (`||` instead of the previous `&&` operator). I added that change to support uploading `heic` images as we can't display thumbnails for them at the moment. I will try to come up with a clean solution to this.
